pom.xml

```diff
@@ -382,7 +403,7 @@
                     <header><![CDATA[<br>Jakarta Activation API v${project.version}]]></header>
                     <bottom>
                         <![CDATA[
-Comments to : <a href="mailto:jaf-dev@eclipse.org">jaf-dev@eclipse.org</a>.<br>
+Comments to : <a href="mailto:angus-dev@eclipse.org">angus-dev@eclipse.org</a>.<br>
 Copyright &#169; 2019, ${current.year} Eclipse Foundation. All rights reserved.<br>
 Use is subject to <a href="{@docRoot}/doc-files/speclicense.html" target="_top">license terms</a>.]]>
```

Comment:
drop this "use" sentence, this is not spec project and does not contain "jakarta" packages

Reply:
Fixed. speclicense.html file was removed from the project.